aws/auto-scaling-group/vpc.tf

```diff
@@ -19,21 +20,22 @@ resource "aws_internet_gateway" "igw" {
   })
 }
 
-resource "aws_subnet" "subnets" {
+# Private subnets in which the instances get private IP addresses.
+resource "aws_subnet" "instances_subnets" {
   count = length(var.availability_zones)
 
   vpc_id                  = aws_vpc.vpc.id
   cidr_block              = cidrsubnet(var.vpc_cidr_block, 4, count.index)
   availability_zone       = var.availability_zones[count.index]
   map_public_ip_on_launch = var.public_subnets # Default is false. 
```

Comment:
Not the correct line since it was not changed in this PR. But what I noticed when using this module is that when we pass a "key_name" for ssh access into the ec2 instances it does not conditionally create a security rule to allow for incoming ssh traffic. Could you add this? Should be somewhat small. If not perhaps add a ticket/issue. Thanks!

Reply:
Great idea! I added the ingress rule for SSH in this PR :D